docs/src/docker.md

```diff
@@ -13,22 +13,38 @@ This image is published on [Docker Hub].
 
 ### Pull the image
 
-```bash js python csharp
```

Comment:
Should we support multiple languages in the filter instead?

Reply:
we'll have dedicated images soon - so the next one who needs it adds it.